tests/components/alexa/test_capabilities.py

```diff
@@ -756,3 +757,25 @@ async def test_report_image_processing(hass):
         "humanPresenceDetectionState",
         {"value": "DETECTED"},
     )
+
+
+async def test_get_property_blowup(hass, caplog):
+    """Test we handle a property blowing up."""
+    hass.states.async_set(
+        "climate.downstairs",
+        climate.HVAC_MODE_AUTO,
+        {
+            "friendly_name": "Climate Downstairs",
+            "supported_features": 91,
+            climate.ATTR_CURRENT_TEMPERATURE: 34,
+            ATTR_UNIT_OF_MEASUREMENT: TEMP_CELSIUS,
+        },
+    )
+    with patch(
+        "homeassistant.components.alexa.capabilities.float",
+        side_effect=Exception("Boom Fail"),
+    ):
+        properties = await reported_properties(hass, "climate.downstairs")
+        properties.assert_not_has_property("Alexa.ThermostatController", "temperature")
+
+    assert "Boom Fail" in caplog.text
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6090606)